### ISSUE TITLE
At some angles, the material 'disappear'

### ISSUE THREAD

**Issue by FutureFireplace** [playcanvas/engine#4269]
While orbiting, there seem to some kind of bug, in relation to above-and-below 180 degrees as threshold.
As such; at some angles, the material 'disappear'?

Video example: https://www.youtube.com/watch?v=JJbjtlTFTWA

Previously, from time to time, If have been subject to the same issue at other projects.

I am using Chrome browser

![image](https://user-images.githubusercontent.com/6838800/169882032-41896793-33f4-4b9f-90d1-e97c02c04819.png)

CF: https://github.com/playcanvas/engine/issues/4268 as well

**Comment by yaustar** [comment]
Off hand from the video, it looks like a transparency rendering order issue. Difficult to tell without having a build to look at. Would you be able to share a build link?

**Comment by FutureFireplace** [comment]
From the two issues (this and https://github.com/playcanvas/engine/issues/4268), I will focus on this and close the other for now, as that for sure will require sharing-the-open-project + this minor one might help the larger (if lucky).

Ok, the 'Layers' area of the engine+editor may appear quite simple to most developers, but I am not sure I fully understand the inner workings 100% (do get that 'Transparency' is needed to execute 'opacity effect', and for some months ago, I got the 'assignment of layers to objects'-routine).

But I am still not fully into 'how the layers' (at set in the editor/settings) inflict on each other, in regards settings-order.
For instance - here is how I have set the various layers in the editor in my project -> will two consecutive 'Transparency' layers affect the render (and the same in regards to 'two consecutive Opaque Layers')?

![image](https://user-images.githubusercontent.com/6838800/169974662-a2ce07bd-1898-4aa4-ba58-ac7122f0136f.png)

PS: As this is no longer a direct 'Bug issue', I am partially transferring it to the Forum: https://forum.playcanvas.com/t/layers-and-affect-of-the-order/25855
